### ISSUE TITLE
haml-lang.com is down, update readme

### ISSUE THREAD

**Issue by Incubatio** [creationix/haml-js#93]
You might want to remove haml-lang.com from your readme.

Could be nice to get that website back on and host it as a github page.

Here is the last snapshot of the website on web.archive
https://web.archive.org/web/20150513150408/http://haml.info/

**Comment by frob** [comment]
It must be some sort of github thing. Those links are just ```[haml][]``` in the markdown.

**Comment by frob** [comment]
Nope, never new you could just add a bunch of link tokens at the bottom of a page for repeated links. Very cool.

**Comment by frob** [comment]
This can be closed now.
